core/java/iesi-rest-without-microservices/src/main/java/io/metadew/iesi/server/rest/script/dto/ScriptDtoRepository.java

```diff
@@ -37,6 +37,8 @@
 
     private final MetadataRepositoryConfiguration metadataRepositoryConfiguration;
     private final FilterService filterService;
+    private boolean fetchAllScript = false;
+    private boolean isOnlyInactive = false;
```

Comment:
why are these needed?

Reply:
These flag are being used while fetching Deleted and Active Script.